layouts/partials/search.html

```diff
@@ -0,0 +1,88 @@
+<link href="/pagefind/pagefind-ui.css" rel="stylesheet">
+<script src="/pagefind/pagefind-ui.js"></script>
+
+<style>
+  .search-button {
+      border-radius: 20px;
+      padding: 8px;
+      margin-right: 15px;
+      cursor: pointer;
+  }
+  .search-dialog {
+      padding: 15px;
+      width: 80%;
+      border-radius: 1rem;
+  }
+  .search-dialog::backdrop {
+    background-color: rgb(0, 0, 0, 0.5);
+    backdrop-filter: blur(3px);
+   }
+  .search-dialog input {
+      padding: 10px 15px;
+      color: #333;
+  }
+  .pagefind-ui button {
+      border: none;
+  }
+  .pagefind-ui__result-title.svelte-4xnkmf .pagefind-ui__result-link.svelte-4xnkmf {
+      color: unset;
+  }
+  input.pagefind-ui__search-input.svelte-e9gkc3 {
+      color: unset;
+  }
+</style>
+
+<dialog class="search-dialog"></dialog>
+
+<script>
```

Comment:
I tried but wasn't able to resolve (404) the standalone js file.  I put it in `assets/js/search.js` with the URL "./js/search.js", following the pattern of CSS files under theme-css, but no luck.  Also tried adding and putting it under a dedicated `assets/theme-js` directory, so it would look more identical to the CSS case but that didn't work either.

So I'm inclined to leave the JS here unless we can figure out how to get hugo to serve it in a way that makes sense.  But it doesn't seem important enough to spend a lot of time on.

Reply:
Ok, I found the js bundling stuff, so I see no need to explicitly link to the js in the html as it will get swept into the bundle automatically.  Went ahead and separated it out into search.js.